examples/src/main/java/com/mapbox/navigation/examples/core/SimpleMapboxNavigationKt.kt

```diff
@@ -305,6 +305,7 @@ class SimpleMapboxNavigationKt :
         override fun onElectronicHorizonUpdated(horizon: EHorizon, type: String) {
             Timber.d("DEBUG EH horizon=$horizon")
             Timber.d("DEBUG EH type=$type")
+            Timber.d("DEBUG EH mpp=${horizon.mpp()}")
```

Comment:
Noticed the puck a bit janky when testing, not sure if caused by this or just a coincidence. We should investigate further 🕵️‍♂️